src/Functions/tokens.cpp

```diff
@@ -18,12 +18,75 @@ namespace ErrorCodes
     extern const int ILLEGAL_COLUMN;
 }
 
+namespace
+{
+template <typename T>
+std::optional<T> getArgument(const ColumnsWithTypeAndName & arguments, size_t argument_index, std::string_view function_name = "")
+{
+    static_assert(std::same_as<T, std::string_view> || std::same_as<T, UInt64> || std::same_as<T, std::vector<String>>);
+
+    if (argument_index >= arguments.size())
```

Comment:
If `argument_index > arguments.size()` is not a sign of a potential error? If tho, then probably it is better to trigger some exception:

> Errors should never pass silently.
> Unless explicitly silenced.